src/util/options.schema.json

```diff
@@ -966,6 +966,15 @@
           "enum": ["ast", "cfg"],
           "default": "ast"
         },
+        "ignore-const-args": {
+          "title": "incremental.ignore-const-args",
+          "description": "List of functions for which constant parameters are not compared",
```

Comment:
Might be good to have this description say that this is useful for `__LINE__` constants.